we-dance/platform
# i18n: auth

closes  #73 
## How to test it: 
https://wedance-stefanacef.netlify.app/
- Test the pop-up: Log out from your account, and search for a dancer's name and on the bottom you will see the watch button and star button and the three dots on the left, push on them, and the pop-up will appear
![image](https://user-images.githubusercontent.com/71949222/152438253-2c7702d5-8d31-4598-800c-006c63889e74.png)
- The no password page: go on no password page, write your email and press the button and a pop up will appear
